Portal/vip-application/src/main/java/fr/insalyon/creatis/vip/application/server/business/WorkflowExecutionBusiness.java

```diff
@@ -54,11 +56,17 @@
     private static final Logger logger = Logger.getLogger(WorkflowExecutionBusiness.class);
     private WorkflowEngineInstantiator engine;
 
-    public WorkflowExecutionBusiness(String applicationClass) throws BusinessException {
+    public WorkflowExecutionBusiness(String engineEndpoint) throws BusinessException {
 
-        engine = WorkflowEngineInstantiator.create(applicationClass);
-    }
+        //HACK for testing while still having simulations launched with VIP 1.16.1; to be removed before getting in production or replaced with a proper constant
+        if(engineEndpoint == null){
+            logger.info("WorkflowExecutionBusiness, endpoint is null, setting it to http://data-manager.grid.creatis.insa-lyon.fr/cgi-bin/m2Server-gasw3.1/moteur_server");
+            engineEndpoint="http://data-manager.grid.creatis.insa-lyon.fr/cgi-bin/m2Server-gasw3.1/moteur_server";
+        }
```

Comment:
OK but this code will never be used during testing phase, and it won't be used in production either. I'm wondering if we should have a fallback for these cases where no engine is declared, or if we just want to make sure that this situation doesn't happen (we could raise a proper error message when it does). Thinking about it, I'm not very keen about a fallback.